ui/src/utils/utils.tsx

```diff
@@ -36,3 +36,17 @@ export const isFeature = (featureType: string) => {
     featureType === FeatureType.DerivedFeature
   );
 };
+
+export const enum LocalVariables {
+  PreKeyword = "preKeyword",
+  PreProject = "preProject",
+}
+
+export const setLocalVariables = (preProject = "", preKeyword = "") => {
```

Comment:
how about we make project and key word as url query string

Reply:
Not very sure if I get your point. Since feature list is a component of the feature page, I didn't find a way to pass project and keyword as a part of url to it from another page directly. I applied url in my first commit but I don't think it's a good solution.